wolox-books/src/app/screens/auth/screens/login/login.component.ts

```diff
@@ -0,0 +1,39 @@
+import { Component, OnInit } from '@angular/core';
+import { FormGroup, FormBuilder, Validators } from '@angular/forms';
+import IUser from 'src/app/models/user.interface';
+import IFormItem from '../../models/form-item.interface';
+import { PasswordValidationService } from 'src/app/validators/password-validation.service';
+import { LOGIN_ITEMS } from './login-constants';
+
+@Component({
+  selector: 'app-login',
+  templateUrl: './login.component.html',
+  styleUrls: ['./login.component.scss'],
```

Comment:
Delete login scss file and just reference the auth component.

Reply:
Done 🚀